apps/competition-service/Dockerfile

```
@@ -0,0 +1,25 @@
+# 루트 빌드 설정 파일 복사
+COPY build.gradle settings.gradle gradlew ./
+COPY gradle ./gradle
+
+# common-libs 복사
+COPY common ./common
+
+# competition-service 소스 복사
+COPY apps/competition-service ./apps/competition-service
+
+RUN chmod +x gradlew
+RUN ./gradlew :apps:competition-service:clean :apps:competition-service:build -x test --no-daemon
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)